tasks/broad/ImputationBeagleTasks.wdl

```diff
@@ -122,7 +122,7 @@ task Impute {
     Int start                # not needed if ref file has been chunked and you are using the entire chunk
     Int end                  # not needed if ref file has been chunked and you are using the entire chunk
 
-    String beagle_docker = "us-central1-docker.pkg.dev/morgan-fieldeng-gcp/imputation-beagle-development/imputation-beagle:0.0.1-01Mar24.d36-wip-temp-20240301"
+    String beagle_docker = "us.gcr.io/broad-gotc-prod/imputation-beagle:0.0.2-17Dec24.224-1739986611"
```

Comment:
note will need to update this again when we confirm that this docker is what we want - will update to version 1.0.0 instead of 0.0.X